test/e2e/flowaggregator_test.go

```diff
@@ -935,6 +935,8 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 		cmdStr += fmt.Sprintf(" -p %d", iperfSvcPort)
 	}
 	timeNow := time.Now()
+	// wait 2 seconds to mitigate the possibility that there is a time difference between the K8s Nodes.
+	time.Sleep(2 * time.Second)
```

Comment:
if this is an actual possibility that can impact tests, we should come up with a better solution. I assume that for CH, we could drop all rows in the table in-between tests (and stop the time filtering). We could have a similar command for the IPFIX collector?

Reply:
Deleting all rows in ClickHouse is feasible. But I think that's not doable for the IPFIX collector as we are using `kubectl logs` to get the logs? In previous PR, I was thinking of restarting the ClickHouse and IPFIX collector. But we need to restart the Flow Aggregator if we create a new IPFIX collector, largely increasing the test time as we need to recreate them in all the subtests. After discussing this with Yongming, I was thinking it's better and easier to recreate iperftest Pods.
However, there is no guarantee that the IPs will be different when we recreate the Pods as you mentioned (But I think it is pretty rare that the IP will be reused in the e2e test)?
What do you think?

Reply:
> But I think that's not doable for the IPFIX collector 

I guess there is still the option of ignoring the first N lines of the logs, where N is the length of the logs at the beginning of the test.

> But I think it is pretty rare that the IP will be reused in the e2e test

Yes, in practice the host-local IPAM will mostly prevent re-use

Was there another concern about re-creating the iperf Pods increased related to increased test time?

If IP reuse was the main "concern", we can always randomize the Pod name to achieve the same uniqueness right?

Reply:
> Was there another concern about re-creating the iperf Pods increased related to increased test time?
The time required for recreating the iperf Pods has seen a smaller increase compared to the time needed for recreating IPFIX/CH/FA/FE.
For recreating the iperf Pods, the test time increases from 20m to 25m.
For recreating IPFIX/CH/FA/FE Pods, the test time increases from 20m to 27m.

I'm thinking why don't we just add labels to the iperf Pods in every subtests. We are able to use the label to filter records and it takes much shorter time to modify the labels.